src/directory.rs

```diff
@@ -20,9 +17,7 @@ pub(crate) fn get_file_iter(
             match mimetype::image_or_video(&f_path) {
                 Ok(true) => v.push(PathBuf::from(f_path)),
                 Ok(false) => {}
-                Err(_) => {
-                    eprintln!("skip {}", &f_path.display())
-                }
+                Err(_) => {}
```

Comment:
I think we should log the file path and the error here. I haven't seen the implementation of `image_or_video` but I am assuming this is a valid error, is my assumption correct?

Reply:
I placed back the eprintln, atm `mimetype::image_or_video` has a bug that will get it's own issue and then we can change and log it there.